### DISCUSSION TITLE
getValues('foo') returns whole form object instead {'foo':'bar'}

### DISCUSSION THREAD

**Discussion by sbimochan** [react-hook-form/react-hook-form#2110] [General] [answered]
getValues('foo') returns whole form object instead {'foo':'bar'}

**Comment by pmaier983** [comment, marked as answer]
Have you seen [This CodeSandbox](https://codesandbox.io/s/react-hook-form-v6-getvalues-v2gc7)? If this doesn't help you please create a CodeSandbox detailing your issue.

**Reply by bluebill1049** [reply]
Thanks @pmaier983 for helping questions :)

**Reply by pmaier983** [reply]
There will soon be too many for one man to manage 😬. It takes a team 😄

**Reply by bluebill1049** [reply]
Yeah, sometimes 1 hr is not enough for me now, it used to be 30mins every morning.
